### PR TITLE
Feature/#155111485

### PR DESCRIPTION
**Pull request by goodgoof**
#### Pre-Submission Checklist:

- [x] Your pull request targets the `master` branch of calltocode.org
- [x] Your branch is updated with the latest changes on `master`
- [x] Your branch passes linting, building and all new and existing tests with command `yarn test`
- [x] Your branch passes the end-to-end tests with command `yarn e2e`
- [x] The story that this pull request addresses is referenced above by ID

#### Post-Submission Checklist:

- [ ] Your branch has no conflicts with `master` (find this out at the bottom of your pull request)
- [ ] Address any reviews on your pull request, as it will be unable to be merged until it is approved

#### Description of Changes:

Adding the Landing page Section F. Below is the screenshot. I have fixed the spacing between projects and about. Thanks

screen shot 2018-03-18 at 3 56 23 pm

### CODE REVIEW

File: client/components/LandingF/LandingF.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+        <div className={styles.column3}>
+          <ul className={styles.list}>
+            <li className={styles.about}><a href="#" className={styles.link}>About</a></li>
+          </ul>
+        </div>
+      </div>
+    )
+  }
+};
```
Comment (timdavish) [inline client/components/LandingF/LandingF.js]: this semicolon isn't needed

File: client/components/LandingF/LandingF.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  padding-left: 0px;
+  text-align: left;
+  width: 131px;
+}
+
+.column3 {
+  margin-left: 75px;
+  width: 131px;
+}
+//
```
Comment (timdavish) [inline client/components/LandingF/LandingF.scss]: go ahead and remove these comments

Comment (goodgoof) [reply client/components/LandingF/LandingF.scss]: removed the comments.

File: client/components/LandingF/LandingF.js (hunk 1 above)
Comment (goodgoof) [reply client/components/LandingF/LandingF.js]: removed semicolon

**Pull request merged**